src/Transformers/OneHotEncoder.php

```diff
@@ -42,11 +42,25 @@ class OneHotEncoder implements Transformer, Stateful, Persistable
     protected ?array $categories = null;
 
     /**
-     * Return the data types that this transformer is compatible with.
+     * The categories that should be ignored
      *
-     * @internal
+     * @var array<string>
+     */
+    protected array $drop = [];
+
+    /**
+     * @param string|array $drop The categories to drop during encoding
+     */
+    public function __construct($drop = [])
```

Comment:
Although I think it's nice to be liberal with the type of this argument, it's tradition to prefer strict types unless necessary. In this case it's not necessary though.